src/app.js

```diff
@@ -76,4 +83,24 @@ var HackToolboxApplication = GObject.registerClass(class extends Gtk.Application
         this._windows[busName][objectPath].present();
         this.release();
     }
+
+    _onFlipBack(action, parameterVariant) {
+        this.hold();
+        const unpacked = parameterVariant.deep_unpack();
+        const [busName, objectPath] = unpacked;
+        log(`Call flip-back with ${JSON.stringify(unpacked)}`);
+
+        const win = this._windows[busName][objectPath];
+        if (win) {
```

Comment:
In the case this check fails, we won't call `release()`.

Reply:
Fixed below.